exporter/templates_test.go

```diff
@@ -835,14 +905,14 @@ func printXmlString(xmlString string) {
 
 func xmlRootNodeForQrdaOid(qrdaOid string) *xml.ElementNode {
 	fileName := "_" + qrdaOid + ".xml"
-	printXmlString(generateXML(fileName, getDataForQrdaOid(qrdaOid)))
+	//printXmlString(generateXML(fileName, getDataForQrdaOid(qrdaOid)))
 	return xmlRootNode(generateXML(fileName, getDataForQrdaOid(qrdaOid)))
 }
 
 // same as xmlRootNodeForQrdaOid() function but allows custom input data (should be an EntryInfo struct)
 func xmlRootNodeForQrdaOidWithData(qrdaOid string, data interface{}) *xml.ElementNode {
 	fileName := "_" + qrdaOid + ".xml"
-	printXmlString(generateXML(fileName, data))
+	//printXmlString(generateXML(fileName, data))
```

Comment:
Generally we leave them there so it's a readily available (and readably easy) option to uncomment them for debugging. But I think we generally try to have the master branch have the commented versions, which is why I did that.